lighthouse-core/report/html/renderer/report-ui-features.js

```diff
@@ -157,6 +166,25 @@ class ReportUIFeatures {
     this._copyAttempt = false;
   }
 
+  _setupMetricDescriptionToggleElements() {
+    this.metricDescriptionToggleEl =
+      /** @type {HTMLInputElement} */ (this._dom.find('.lh-metrics-toggle input', this._document));
+    this.metricAuditGroup = this._dom.find('.lh-audit-group--metrics', this._document);
+  }
+
+  _setupMetricDescriptionToggleListeners() {
+    const toggle = () => this.metricDescriptionToggleEl.click();
+    this.metricDescriptionToggleEl.addEventListener('input', this._toggleMetricDescription);
+    for (const el of this._dom.findAll('.lh-metric', this._document)) {
```

Comment:
event delegation will be cleaner here.

Reply:
hm?

Reply:
we really only need one click listener, then just make sure the target was an `.lh-metric` (though it might as well be on the entire section considering how much of it `lh-metric`s take up...I think I agree with @paulirish about the clicking thing)

Reply:
here's event delegation. and also only using the metric titles as click targets.

```js

    this.metricAuditGroup.addEventListener('click', e => {
      if (e.target.closest('.lh-metric__title')) this.metricDescriptionToggleEl.click()
    });
```